server/protocols/jmap-integration-testing/jmap-integration-testing-common/src/test/java/org/apache/james/jmap/methods/integration/SetMailboxesMethodTest.java

```diff
@@ -387,7 +388,14 @@ public void subscriptionUserShouldBeChangedWhenCreateThenUpdateMailboxNameWithJM
             .body(requestBody)
             .post("/jmap");
 
-        assertThat(mailboxProbe.listSubscriptions(username)).containsOnly("mySecondBox");
+        List<String> expectedMailboxes = ImmutableList.<String> builder()
+            .add(DefaultMailboxes.OUTBOX)
+            .add(DefaultMailboxes.SENT)
+            .add(DefaultMailboxes.TRASH)
+            .add(DefaultMailboxes.DRAFTS)
+            .add("mySecondBox")
+            .build();
+        assertThat(mailboxProbe.listSubscriptions(username)).containsOnlyElementsOf(expectedMailboxes);
```

Comment:
you probably don't need to build the list in the first place. Why don't you use containsOnly(...) instead ?